api/endpoints/user.md

```diff
@@ -122,10 +167,165 @@ The following attributes are required to create a User: `firstName`, `lastName`,
 
     [User][]
 
-## Get current logged in user [/users/me]
+### Get current logged in user [/users/me]
 
 Get data for currently logged in user
 
-### Hold a Request [GET]
+
+### Retrieve Current User [GET]
 + Response 200
     [User][]
+
+## Get availability of a username [/users/username/{{username}]
+
+Get boolean value if {username} already exists
+
+
+### Check if Username Exists [GET]
+
++ Response 200 (application/json)
+
+            {
+                "available": "true",
+            }
+
+## Get Current User Report Auth Data [/users/me/reportAuth]
+
+Get current user report auth data
+
+### Retrieve Current User Report Auth Data [GET]
+
++ Response 200 (application/json)
+
+            {
+                "server:" "http://andross.corp.iofficecorp.com",
+                "authToken:" "a234893fmdfjhdkjh345897djhdjkh59",
+                "orgId:" "dev_fmcloud",
+                "username: "foxmccloud"
```

Comment:
@rmoscowitz lol i thought this was fake 😄 I just typed a bunch of random stuff for authToken, and followed the Starfox references for the other stuff, including the server name.